simple_history/manager.py

```diff
@@ -34,6 +34,11 @@ def get_queryset(self):
             key_name = self.instance._meta.pk.name
         return self.get_super_queryset().filter(**{key_name: self.instance.pk})
 
+    def get_excluded_fields(self):
+        if isinstance(self.instance._meta.model.history, self.__class__):
```

Comment:
Don't want to hardcode `history` here. Instead, use `utils.get_history_manager_for_model`

Reply:
What is this conditional checking for?

Reply:
I was using this conditional check because I thought all the instances might not have a model history class.

Reply:
They should all have a model history class. However, they might not have `_history_excluded_fields` so you should do:
`excluded_fields = getattr(self.model, '_history_excluded_fields', [])`

Reply:
oh shoot you're right. Now that I'm looking at the print statements for debugging it's painfully obvious. Damn!